src/extern/wxWidgets/listctrl.cpp

```diff
@@ -4093,7 +4093,7 @@ int wxListMainWindow::GetSelectedItemCount() const
     if ( IsSingleSel() )
         return HasCurrent() ? IsHighlighted(m_current) : false;
 
-    // virtual controls remmebers all its selections itself
+    // virtual controls remembers all its selections itself
```

Comment:
I noticed `virtual controls` is plural, so, maybe is better `remember` instead `remembers` ?

Reply:
@sc0w then you'd have to re-evaluate using `its` and `itself`, right?

Reply:
no problem, squashed and merged